.github/workflows/ci.yml

```diff
@@ -65,4 +66,114 @@ jobs:
 
       - name: Run tests
         id: run-tests
-        run: sbt ++${{ env.scala }} test
\ No newline at end of file
+        run: sbt ++${{ env.scala }} test
+
+  ci-docker:
+    # run on external PRs, but not on internal PRs since those will be run by push to branch
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check-out repository
+        id: repo-checkout
+        uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: 11
+
+      - name: Cache SBT
+        id: cache-sbt
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.coursier
+          key: ${{ runner.os }}-sbt-${{ env.scala }}-${{ hashFiles('**/build.sbt') }}
+
+      - name: Compile
+        run: sbt compile
+
+      - name: Build jvm docker image
+        run: sbt "project server; assembly; Docker / publishLocal"
+
+      - name: Build native docker image
+        run: sbt "project nativeServer; assembly; GraalVMNativeImage / packageBin; Docker / publishLocal"
+
+  publish:
```

Comment:
shouldn't we have a dependency here so that this is only run if ci-* jobs pass?

Reply:
it's temporarily removed to speed up testing